setup.py

```diff
@@ -40,12 +45,17 @@
 ################################################################################
 # Actual setup.py content
 ################################################################################
-from itertools import chain
 
-from setuptools import setup
-from setuptools.config import read_configuration
-
-from extension_helpers import get_extensions
+VERSION_TEMPLATE = """
+# Note that we need to fall back to the hard-coded version if either
+# setuptools_scm can't be imported or setuptools_scm can't determine the
+# version, so we catch the generic 'Exception'.
+try:
+    from setuptools_scm import get_version
+    version = get_version(root='..', relative_to=__file__)
```

Comment:
An interesting suggestion from the astropy discussion here is to change this to:
```suggestion
    __version__ = get_version(root='..', relative_to=__file__)
```

and then update `__init__.py` so it does `from version import __version__`.